index.bs

```diff
@@ -3657,7 +3688,89 @@ This is because the first 37 bytes of the signed data in a FIDO U2F authenticati
 <code>[=authDataExtensions|extensions=]</code> are never present. FIDO U2F authentication signatures can therefore be verified by
 the same procedure as other [=assertion signatures=] generated by the [=authenticatorGetAssertion=] operation.
 
+### Credential Backup State ### {#sctn-credential-backup}
+
+Credential [=backup eligibility=] and current [=backup state=] is conveyed by the `BE` and `BS` [=flags=] in the [=authenticator data=], as 
+defined in <a href="#table-authData">Table <span class="table-ref-previous"/></a>.
+
+The value of the `BE` [=flag=] is set during [=authenticatorMakeCredential=] operation and MUST NOT change.
+
+The value of the `BS` [=flag=] may change over time based on the current state of the [=Public Key Credential Source|credential source=]. <a href="#table-backupStates">Table <span class="table-ref-following"/></a> below defines
+valid combinations and their meaning.
+
+<figure id="table-backupStates" class="table">
+    <table class="complex data longlastcol">
+            <tr>
+            <th>`BE`</th>
+            <th>`BS`</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td>`0`</td>
+            <td>`0`</td>
+            <td>
+                The credential is a [=single-device credential=].
+            </td>
+        </tr>
+        <tr>
+            <td>`0`</td>
+            <td>`1`</td>
+            <td>
+                This combination is not allowed.
+            </td>
+        </tr>
+        <tr>
+            <td>`1`</td>
+            <td>`0`</td>
+            <td>
+                The credential is a [=multi-device credential=] and is not yet backed up.
+            </td>
+        </tr>
+        <tr>
+            <td>`1`</td>
+            <td>`1`</td>
+            <td>
+                The credential is a [=multi-device credential=] and is backed up.
+            </td>
+        </tr>
+    </table>
+    <figcaption>
+        `BE` and `BS` [=flag=] combinations
+    </figcaption>
+</figure>
+
+It is RECOMMENDED that [=[RPS]=] store the most recent value of these [=flags=] with the [=user account=] for future evaluation.
+
+The following is a non-normative, non-exhaustive list of how [=[RPS]=] might utilize these [=flags=]:
+
+    - Requiring additional [=authenticators=]:
 
+        When `BE` [=flag=] is set to `0`, the [=generating authenticator=] will never allow the credential to transition from a 
```

Comment:
(not blocking) Does any credential &ldquo;transition from a [=single-device credential=] to a [=multi-device credential=]&rdquo;? The definition above says that a credential is exactly one of those things, fixed at creation time, unchanging. I fear that this paragraph suggests otherwise.